### ISSUE TITLE
CelParseError: Given string is not a valid CEL expression

### ISSUE THREAD

**Issue by jbalatero** [ChromeGG/cel-js#30]
This code throws an error `const result = evaluate('install == "confirmed"', { install: "confirmed" });`

Error

```
CelParseError: Given string is not a valid CEL expression: Expecting: one of these possible Token sequences:
  1. [OpenParenthesis]
  2. [BooleanLiteral]
  3. [Null]
  4. [StringLiteral]
  5. [Float]
  6. [Integer]
  7. [ReservedIdentifiers]
  8. [OpenBracket]
  9. [OpenCurlyBracket]
  10. [MacrosIdentifier]
  11. [Identifier]
but found: 'in'
```

But not this if you remove the `in` like this `const result = evaluate('stall == "confirmed"', { stall: "confirmed" });`

it works.

**Comment by ChromeGG** [comment]
Thanks for sharing, I'll take a look in this week 👍

**Comment by ChromeGG** [comment]
Fixed in `v0.2.1`. Thanks for finding this bug and reporting it!
